Code2Gether-Discord-Bot.Library/BusinessLogic/GitHubJoinOrgLogic.cs

```diff
@@ -41,11 +41,15 @@ public override async Task<Embed> ExecuteAsync()
             return embed;
         }
 
-        private async Task<EmbedContent> JoinGitHubOrganization()
+        private async Task<HttpResponseMessage> PostNewInviteToOrganization()
         {
-            var githubClient = new GitHubClient(_githubAuthToken);
-            var response = await githubClient.InviteViaEmailToOrganizationAsync(_githubOrganizationName, _userGitHubEmail);
-            return ParseHttpResponseToEmbedContent(response);
+            var githubClient = new GitHubClient(_githubAuthToken, _githubOrganizationName);
+
+            var json = $"{{\"email\":\"{_userGitHubEmail}\"}}";
```

Comment:
I think we maybe should avoid this "magic string" and start to use `Json.SerializeObject` to return this json string.
```csharp
var data = new Dictionary<object, object>
{
   ["email"] = "sample@example.com"
};

var json = JsonConvert.SerializeObject(data);
```

Reply:
Good suggestion! I'll look into updating it. Feel free to mark your review as "request changes"

Reply:
> Feel free to mark your review as "request changes"

One of my first times reviewing a PR, so I did it all wrong LFMAO